modules/custom/d_content/config/install/user.role.d_editor.yml

```diff
@@ -9,6 +9,7 @@ permissions:
   - 'access content overview'
   - 'access taxonomy overview'
   - 'access toolbar'
+  - 'access contextual links'
```

Comment:
It would be cool to include an update here for existing sites :-)
You can find the update guide in Droptica's documentation of Droopler